Lib9c/Action/HackAndSlash.cs

```diff
@@ -28,6 +29,7 @@ public class HackAndSlash : GameAction
         public List<Guid> foods;
         public int worldId;
         public int stageId;
+        public int randomBuffId;
```

Comment:
This field can to be nullable.

Reply:
That's right, so if the field is empty or incorrect, it's going to operate to select the best buff on our own.
It also does not consider the wave clear count when creating or updating a state.